src/class-settings.php

```diff
@@ -60,38 +62,45 @@ public function register_fields() {
 						]
 					),
 					'footer_settings' => new \Fieldmanager_Group(
+						// @phpstan-ignore-next-line the Fieldmanager doc block is incorrect.
 						[
 							'label'       => __( 'Footer Settings', 'wp-newsletter-builder' ),
 							'collapsed'   => true,
 							'collapsible' => true,
 							'children'    => [
 								'facebook_url'  => new \Fieldmanager_Link(
+									// @phpstan-ignore-next-line the Fieldmanager doc block is incorrect.
```

Comment:
Is this ticketed as an issue on GH? we may want to

Reply:
That's a good idea. I just created an issue: https://github.com/alleyinteractive/wordpress-fieldmanager/issues/867

TY :)